docs-source/content/samples/simple/domains/domain-home-in-image/_index.md

```diff
@@ -145,11 +147,12 @@ The following parameters can be provided in the inputs file.
 | `imagePullSecretName` | Name of the Kubernetes secret to access the Docker Store to pull the WebLogic Server Docker image. The presence of the secret will be validated when this parameter is specified. |  |
 | `includeServerOutInPodLog` | Boolean indicating whether to include `server.out` to the pod's stdout. | `true` |
 | `initialManagedServerReplicas` | Number of Managed Servers to initially start for the domain. | `2` |
-| `javaOptions` | Java options for starting the Administration and Managed Servers. A Java option can have references to one or more of the following pre-defined variables to obtain WebLogic domain information: `$(DOMAIN_NAME)`, `$(DOMAIN_HOME)`, `$(ADMIN_NAME)`, `$(ADMIN_PORT)`, and `$(SERVER_NAME)`. | `-Dweblogic.StdoutDebugEnabled=false` |
+| `javaOptions` | Java options for starting the Administration and Managed Servers. A Java option can have references to one or more of the following pre-defined variables to obtain WebLogic domain information: `$(DOMAIN_NAME)`, `$(DOMAIN_HOME)`, `$(ADMIN_NAME)`, `$(ADMIN_PORT)`, and `$(SERVER_NAME)`. If `sslEnabled` is set to `true` and the WebLogic Demo certificate is used, add `-Dweblogic.security.SSL.ignoreHostnameVerification=true` to allow the managed servers to connect to the Admin server while booting up.  The WebLogic generated Demo certificate in this environment typically contains a host name that is different from the runtime container's.  | `-Dweblogic.StdoutDebugEnabled=false` |
```

Comment:
Why is "Demo" capitalized?  What do you mean by the host name of the runtime container?  Do you mean the pod's host name or what `hostname` reports inside a container?  I want to understand what you are communicating.

Reply:
Should be lower case, demo certificate, unless referring a specific demo certificate, such as the Demo Certificate Authority X.509 certificate.

Reply:
I'll change it to "demo".  By "the host name of the runtime container", I mean the output of `hostname` or echo $HOSTNAME in a running container.  Should I use "the pod's host name" instead of "the runtime container's host name"?